plugins/modules/packaging/language/cargo.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Radek Sprta <mail@radeksprta.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: cargo
+short_description: Manage Rust packages with cargo
+description:
+  - Manage Rust packages with cargo.
+author: "Radek Sprta (@radek-sprta)"
+options:
+  name:
+    description:
+      - The name of a Rust package to install.
+    type: str
+    required: true
+  path:
+    description:
+      ->
+      The base path where to install the Rust packages. Cargo automatically appends
+      /bin. In other words, /usr/local will become /usr/local/bin.
```

Comment:
```suggestion
      C(/bin). In other words, C(/usr/local) will become C(/usr/local/bin).
```
(See https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation for more details)